.github/workflows/docify.yml

```diff
@@ -0,0 +1,42 @@
+name: 'Docify'
+
+on: [push, pull_request]
```

Comment:
there is a file filter if you want to restrict it to subfolder changes

Reply:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-paths

Reply:
oh nevermind. this is python-type-stubs

Reply:
Well, that's still potentially useful - we should only need to run this if the stubs change, docify changes, the config file changes, or the version of pandas changes. But given we're mostly changing pandas its probably fine to leave as is for now :-)